HardwareDefinitions/avnet_sk_rev2_files_for_SDK_installer/inc/hw/avnet_mt3620_sk_rev2.h

```diff
@@ -0,0 +1,155 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License. */
+
+// This JSON hardware definition file specifies peripheral pinouts
+// of the Rev.2 Avnet MT3620 Starter Kit (SK2). The Rev.2 board can
+// be fitted with an Ethernet Click, it also allows reconfiguration
+// of Click2, Pmod and Grove connectors, by moving config resistors.
+// See Hardware User Guide (pages 22-26) for more details.
+// http://avnet.me/mt3620-kit-UG-V2 
+//
+// ISU1 maps to Click socket #2. It can be configured to support
+// either a 4-wire UART (default) or an SPI interface.
+// UART configuration (default) has R59 and R60 populated with
+// zero ohm resistors (and R61 and R62 unpopulated).
+// SPI configuration requires de-populating R59 and R60, then
+// moving these config resistors to locations R61 and R62.
+//
+// ISU1 maps also to the PMOD connector (unpopulated). It's default
+// configuration supports type 4A UART Pmod peripheral boards for
+// BLE or wired UART expansion. Alternatively it can be configured
+// to support type 2A SPI Pmod peripheral boards.
+// UART configuration (default) has R39 and R50 populated with
+// zero ohm resistors (and locations R41 and R52 unpopulated).
+// SPI configuration requires de-populating R39 and R50, then
+// moving these config resistors to locations R41 and R52.
+//
+// ISU2 maps to the Grove connector. It�s default configuration
```

Comment:
It�s - typo?

Reply:
I've replaced this character with a valid " ' " character, regenerated the *.h file and pushed the changes to my branch.